composer.json

```diff
@@ -72,7 +72,8 @@
 		"npm-asset/textcomplete": "^0.18.2",
 		"npm-asset/typeahead.js": "^0.11.1",
 		"kornrunner/blurhash": "^1.2",
-		"psr/clock": "^1.0"
+		"psr/clock": "^1.0",
+		"enyo/dropzone": "^5.9"
```

Comment:
Please move this alphabetically after the `"divineomega/password_exposed": "^2.8",` line